src/app/wallet/view.nim

```diff
@@ -149,6 +149,13 @@ QtObject:
       self.setCurrentAccountByIndex(0)
     self.accountListChanged()
 
+  proc getFiatValue*(self: WalletView, cryptoBalance: string, cryptoSymbol: string, fiatSymbol: string): string {.slot.} =
+    let val = self.status.wallet.getFiatValue(cryptoBalance, cryptoSymbol, fiatSymbol)
```

Comment:
Might make sense to rename `wallet.getFiatValue()` to something else, because it can be used to calculate crypto values as well (just need to swap the symbols)

Reply:
Maybe something like `convertValue` might make more sense?

Reply:
This is done in a follow-up commit here: https://github.com/status-im/nim-status-client/pull/703/commits/5d8abc2223902e351bedfbd433e3d3eb2f65ae6a